examples/db-templates/mysql-persistent-template.json

```diff
@@ -0,0 +1,192 @@
+              "from": {
+                "kind": "ImageStreamTag",
+                "name": "mysql:latest"
+              },
+              "lastTriggeredImage": ""
+            }
+          },
+          {
+            "type": "ConfigChange"
+          }
+        ],
+        "replicas": 1,
+        "selector": {
+          "name": "mysql"
+        },
+        "template": {
+          "metadata": {
+            "creationTimestamp": null,
+            "labels": {
+              "name": "mysql"
+            }
+          },
+          "spec": {
+            "containers": [
+              {
+                "name": "mysql",
+                "image": "mysql",
+                "ports": [
+                  {
+                    "containerPort": 3306,
+                    "protocol": "TCP"
+                  }
+                ],
+                "env": [
+                  {
+                    "name": "MYSQL_USER",
+                    "value": "${MYSQL_USER}"
+                  },
+                  {
+                    "name": "MYSQL_PASSWORD",
+                    "value": "${MYSQL_PASSWORD}"
+                  },
+                  {
+                    "name": "MYSQL_DATABASE",
+                    "value": "${MYSQL_DATABASE}"
+                  }
+                ],
+                "resources": {},
+                "volumeMounts": [
+                  {
+                    "name": "mysql-data",
+                    "mountPath": "/var/lib/mysql/data"
+                  }
+                ],
+                "terminationMessagePath": "/dev/termination-log",
+                "imagePullPolicy": "IfNotPresent",
+                "capabilities": {},
+                "securityContext": {
+                  "capabilities": {},
+                  "privileged": false
+                }
+              }
+            ],
+            "volumes": [
+              {
+                "name": "mysql-data",
+                "persistentVolumeClaim": {
+                  "claimName": "${PVC_NAME}"
+                }
+              }
+            ],
+            "restartPolicy": "Always",
+            "dnsPolicy": "ClusterFirst",
+            "serviceAccount": ""
+          }
+        }
+      },
+      "status": {}
+    }
+  ],
+  "parameters": [
+    {
+      "name": "MYSQL_USER",
+      "description": "database username",
+      "generate": "expression",
+      "from": "user[A-Z0-9]{3}"
+    },
+    {
+      "name": "MYSQL_PASSWORD",
+      "description": "database password",
+      "generate": "expression",
+      "from": "[a-zA-Z0-9]{8}"
+    },
+    {
+      "name": "MYSQL_DATABASE",
+      "description": "database name",
+      "value": "sampledb"
+    },
+    {
+      "name": "PVC_CAPACITY",
+      "description": "database volume capacity, e.g. 512Mi, 2Gi",
+      "value": "512Mi"
+    },
+    {
+      "name": "PVC_NAME",
+      "description": "persistent volume claim name",
+      "generate": "expression",
+      "from": "pvc-[a-z0-9]{8}"
```

Comment:
Who is it conflicting with?

> On Jun 2, 2015, at 10:50 AM, Martin Nagy <notifications@github.com> wrote:
> 
> In examples/db-templates/mysql-persistent-template.json:
> 
> > +    },
> > +    {
> > +      "name": "MYSQL_DATABASE",
> > +      "description": "database name",
> > +      "value": "sampledb"
> > +    },
> > +    {
> > +      "name": "PVC_CAPACITY",
> > +      "description": "database volume capacity, e.g. 512Mi, 2Gi",
> > +      "value": "512Mi"
> > +    },
> > +    {
> > +      "name": "PVC_NAME",
> > +      "description": "persistent volume claim name",
> > +      "generate": "expression",
> > +      "from": "pvc-[a-z0-9]{8}"
> Prefix + random suffix seems best. But yeah, it can't be sampledb-data, that could conflict easily.
> 
> —
> Reply to this email directly or view it on GitHub.
>